data/json/items/containers.json

```
@@ -2817,7 +2817,7 @@
     "pocket_data": [
       {
         "pocket_type": "CONTAINER",
-        "max_contains_volume": "2749 ml",
+        "max_contains_volume": "3029 ml",
```

```suggestion
        "max_contains_volume": "2777 ml",
```
So they can spawn sealed, as verified with Debug Menu -> Info -> Test Item Group -> MRE

---

You are absolutely right. For some reason I decided to increase the container volume by 280ml instead of 28ml. I'll fix that right away.